Terms and Policies

wallento opened the pull request.

imphil commented:
I've rebased it to the current master and added small changes. Some questions remain, however.

imphil commented on site/sitecontent/terms.md:
Should that be a subheader?

wallento replied:
(y)

imphil commented on site/sitecontent/terms.md:
Do we really need to be that specific here? e.g. "redirected to the LibreCores front page" sounds overly specific.

imphil commented on site/sitecontent/terms.md:
website usage surveys?

wallento replied:
I think thats analytics

imphil commented on site/sitecontent/terms.md:
How about all the other information people enter when creating projects?

wallento replied:
Right, will extend the text

imphil commented on site/sitecontent/terms.md:
Well, first of all we use the information to actually show something on the site -- i.e. not just to "improve" the service, but to provide the service at all.

wallento replied:
I think this whole part relates to metadata (analytics) and not to the actual data. Because we don't "collect" it, but users enter it actively. Anyhow, I will add it to be explicit.

wallento replied:
If you have a look at Github (which seems comparable) thats also kind of the tone: https://help.github.com/articles/github-privacy-statement/

imphil commented on site/sitecontent/terms.md:
Google Analytics probably needs to be mentioned here?

imphil commented on site/sitecontent/terms.md:
Is this UK-specific? Or copied from the US, which doesn't apply to us?

wallento commented:
Hi,

I have replaced the Privacy Policies with those from GitHub and made minor adoptions to our service. Beyond that I incorporated some of the feedback by @imphil.

I feel that it is fit to be in place now, but please review the text again.

Cheers,
Stefan

agathver commented on site/sitecontent/terms.md:
What if I register with a name "coolxyzguy" and tomorrow someone becomes famous as "coolxyzguy" in places outside librecores ? Does that apply to that case ?

wallento replied:
I would tend to say it strongly depends. In the case you describe the original user can keep the name. This clause basically protects from people knowingly using misleading names.

agathver commented on site/sitecontent/terms.md:
"machine-generated"

This is very hard to apply to code. Any project using a scaffolding tool, or some fancy tool that produces some code from the spec would be disqualified

wallento replied:
True, it can be misunderstood. Do you think we should remove it or better specify what is meant?

imphil commented on site/sitecontent/terms.md:
This sounds rather broad and possibly unachievable to me. Does this mean: only for content I personally created I need to warrant that it doesn't infringe proprietary rights, or also for content others created? Otherwise, this clause would mean I personally become responsible for code is infringing, for example, patents and which I added to LibreCores.

imphil commented on site/sitecontent/terms.md:
This paragraph could contradict open source licenses, i.e. LibreCores wants to get broader permissions than allowed under the projects source-code license. I guess it boils down to: what's meant by "submitting Content to LibreCores for inclusion in a Project" -- is that only additional metadata people enter into LibreCores, or does it mean the project's source code? We should at least clarify that here, this paragraph is essential.

imphil commented on site/sitecontent/terms.md:
missing header markup?

imphil commented:
@wallento do you think you find time for this in the next couple of weeks?

wallento commented:
Yes, let's finish this up this week!

On December 4, 2017 10:14:43 PM GMT+01:00, Philipp Wagner <notifications@github.com> wrote:
>@wallento do you think you find time for this in the next couple of
>weeks?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/librecores/librecores-web/pull/135#issuecomment-349107737
